docs/reference/schema.md

```diff
@@ -0,0 +1,539 @@
+
+![Screenshot](../img/hzd_fl_kabul.jpg)
+
+| **Required** | **Attribute**           | **Example**     |
+| :----------: | ----------------------- | --------------- |
+|      \*      | Hazard type             | Flood           |
+|      \*      | Analysis type           | Probabilistic   |
+|      \*      | Calculation method      | Simulated       |
+|              | Geographic area         | Kabul           |
+|              | Frequency type          | Return Period   |
+|              | Occurrence probability  | 100 years       |
+|              | Occurrence time (start) | 1958            |
+|              | Occurrence time (end)   | 2001            |
+|              | Occurrence time (span)  | 44 years        |
+|      \*      | Hazard process          | River flood     |
+|      \*      | Unit of measure         | Water depth (m) |
+
+**Earthquake hazard maps for Afghanistan**
+
+Schema attributes for earthquake hazard map related to occurrence probability of an event with return period of  once in 1000 years over Afghanistan. The seismic data catalogue behind the calculation of occurrence probability start from year 800, covering a period of 1200 years. The hazard intensity is measured as Peak Ground Acceleration, expressed in (g).
+
+![Screenshot](../img/hzd_eq_afg.jpg)
+
+| **Required** | **Attribute**           | **Example**   |
+| :----------: | ----------------------- | ------------- |
+|      \*      | Hazard type             | Earthquake    |
+|      \*      | Analysis type           | Probabilistic |
+|      \*      | Calculation method      | Simulated     |
+|              | Frequency type          | Return Period |
+|              | Occurrence probability  | 1000 years    |
+|              | Occurrence time (start) | 800           |
+|              | Occurrence time (end)   | 2001          |
+|              | Occurrence time (span)  | 1200 years    |
+|      \*      | Hazard process          | Ground motion |
+|      \*      | Unit of measure         | PGA (g)       |
+
+
+````
+
+`````
+
+## Exposure
+
+The exposure schema covers a wide variety of data describing structural, infrastructural and environmental asset, population, and socio-economic descriptors, each with relevant attributes for assessing risk from multiple hazards. The schema was developed based on [GEM Taxonomy 2.0](https://wiki.openstreetmap.org/wiki/GED4ALL) to accommodate the most important spatial features commonly employed in risk analysis to identify and estimate exposed value.
+
+```{eval-rst}
+ .. mermaid::
+
+  classDiagram
+      Model -- Asset1
+      Model -- Asset2
+      Model: Category
+      Model: Occupancy
+      class Asset1{
+        Taxonomy code
+        Value type
+        Value unit
+      }
+      class Asset2{
+        Taxonomy code
+        Value type
+        Value unit
+      }
+```
+
+The main features of an exposure dataset are specified by the **exposure model** attributes.
```

Comment:
This whole bit will also need rewritten as it no longer matches the updated data model

Reply:
Agreed, but, as mentioned in the issue description, I think updating the text is best left until the schema updates are done.